tests/llama_stack/safety/conftest.py

```
@@ -48,38 +48,64 @@ def guardrails_orchestrator_ssl_cert(guardrails_orchestrator_route: Route):
         with open(filepath, "w") as f:
             f.write("\n".join(cert_lines))
 
-        return filepath  # noqa: TRY300
+        return filepath
 
     except Exception as e:  # noqa: BLE001
         raise RuntimeError(f"Could not get certificate from {hostname}: {e}")
```

Ruff (0.15.6)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)